CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # REopt Changelog
 
+## Develop
+- Allow Wind tech to be included when `off_grid_flag` is true
+- Add `operating_reserve_required_pct` to Wind struct and incorporate wind into operating reserve constraints
+- Bug fix to constrain dvCurtail in `time_steps_without_grid`
+- Bug fix to report accurate wind ["year_one_to_load_series_kw"] in results/wind.jl (was previously not accounting for curtailed wind)
+
 ## Develop
```

Comment:
note the double ## Develop headers (sure it will get resolved in merge to master branch)